store/postgres/src/relational_queries.rs

```diff
@@ -1646,25 +1646,53 @@ impl<'a> FilterCollection<'a> {
 pub struct SortKey<'a> {
     column: Option<&'a Column>,
     value: Option<&'a str>,
-    direction: EntityOrder,
+    direction: &'static str,
+    /// Entirely skip creating an `order by` clause. The other fields of this
+    /// struct will be ignored if this is `true`
+    omit: bool,
```

Comment:
It's a bit strange to have an "ignore me" field in a struct. Would it be possible to represent this as `Option<SortKey>` in the `FilterQuery`?

Reply:
Doing it that way would require that every caller checks whether they have a `SortKey` or `None`; I went this route to keep the code using `SortKey` more uniform and easier to follow. Another option would be to make `SortKey` a trait with a noop and a actual implementation, but I felt that was taking this a little too far. It is a bit strange, but the strangeness is all inside the `SortKey` implementation.

Reply:
You could have
`
enum SortKey {
  None,
  Key {
   // fields
  }
}
```
and destructure it at the start of every method, returning early in case of `None`.